src/main/java/frc/robot/RainbowDJ.java

```diff
@@ -11,24 +11,26 @@
 
 import edu.wpi.first.wpilibj.Encoder;
 import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.SpeedController;
 import edu.wpi.first.wpilibj.VictorSP;
 
 public class RainbowDJ {
 
+    // ESCs
+    private final SpeedController rainbowDJMotor;
+
     // Encoder declarations
     private final Encoder rainbowDJEncoder;
 
     // Color Sensor
     private final ColorSensorV3 rainbowDJColorSensor;
 
-    // ESCs
-    private final VictorSP rainbowDJMotor;
-
     // Class initializer
-    public RainbowDJ(int rainbowDJMotorPort, int rainbowDJEncoderChannelA, int rainbowDJEncoderChannelB, I2C.Port colorSensorPort) {
-    
+    public RainbowDJ(SpeedController rainbowDJMotorPort, int rainbowDJEncoderChannelA, int rainbowDJEncoderChannelB,
```

Comment:
The `rainbowDJMotorPort` name is wrong for the type `SpeedControler`.

```suggestion
    public RainbowDJ(SpeedController rainbowDJMotor, int rainbowDJEncoderChannelA, int rainbowDJEncoderChannelB,
```